src/main/scala/rocket/Rocket.scala

```diff
@@ -642,7 +642,7 @@ class Rocket(implicit p: Parameters) extends CoreModule()(p)
   csr.io.counters foreach { c => c.inc := RegNext(perfEvents.evaluate(c.eventSel)) }
 
   if (enableCommitLog) {
-    val pc = Wire(SInt(width=xLen))
+    val pc = Wire(UInt(width=xLen))
     pc := wb_reg_pc
```

Comment:
It gives me the following error during the output of dts.
```
[error] (run-main-0) java.lang.reflect.InvocationTargetException
java.lang.reflect.InvocationTargetException                                                                                                                                                                  
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)                                                          
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)        
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)                         
        at util.HasGeneratorUtilities$$anonfun$2.apply(GeneratorUtils.scala:47)
        at util.HasGeneratorUtilities$$anonfun$2.apply(GeneratorUtils.scala:44)
        at chisel3.core.Module$.do_apply(Module.scala:42)
        at chisel3.Driver$$anonfun$elaborate$1.apply(Driver.scala:92)
        at chisel3.Driver$$anonfun$elaborate$1.apply(Driver.scala:92)
        at chisel3.internal.Builder$$anonfun$build$1.apply(Builder.scala:240)
        at chisel3.internal.Builder$$anonfun$build$1.apply(Builder.scala:238)
        at scala.util.DynamicVariable.withValue(DynamicVariable.scala:58)
        at chisel3.internal.Builder$.build(Builder.scala:238)
        at chisel3.Driver$.elaborate(Driver.scala:92)
Caused by: java.util.NoSuchElementException: None.get
        at scala.None$.get(Option.scala:347)
        at chisel3.internal.firrtl.UnknownWidth.get(IR.scala:183)
        at chisel3.core.Data.getWidth(Data.scala:280)
        at util.package$UIntToAugmentedUInt$.sextTo$extension(Package.scala:44)
        at rocket.Rocket.<init>(Rocket.scala:645)
        at coreplex.BaseCoreplexConfig$$anonfun$$lessinit$greater$1$$anonfun$apply$1$$anonfun$applyOrElse$1.apply(Configs.scala:24)
        at coreplex.BaseCoreplexConfig$$anonfun$$lessinit$greater$1$$anonfun$apply$1$$anonfun$applyOrElse$1.apply(Configs.scala:24)
        at rocket.RocketTileModule$$anonfun$18.apply(RocketTiles.scala:129)
        at rocket.RocketTileModule$$anonfun$18.apply(RocketTiles.scala:129)
        at chisel3.core.Module$.do_apply(Module.scala:42)
        at rocket.RocketTileModule.<init>(RocketTiles.scala:129)
        at rocket.RocketTile.module$lzycompute(RocketTiles.scala:116)
        at rocket.RocketTile.module(RocketTiles.scala:116)
        at rocket.RocketTile.module(RocketTiles.scala:26)
java.lang.RuntimeException: Nonzero exit code: 1
        at scala.sys.package$.error(package.scala:27)
        at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:2081)
        at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:2081)
        at scala.Option.foreach(Option.scala:236)
        at sbt.BuildCommon$class.toError(Defaults.scala:2081)
        at sbt.Defaults$.toError(Defaults.scala:39)
        at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$36$$anonfun$apply$37.apply(Defaults.scala:740)
        at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$36$$anonfun$apply$37.apply(Defaults.scala:738)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
        at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:63)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
        at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[error] (rocketchip/compile:runMain) Nonzero exit code: 1
```

I find [similar code here](https://github.com/freechipsproject/rocket-chip/blob/e3e77d68e6b7affc24dca56622ab95b6b5abf55c/src/main/scala/rocket/CSR.scala#L349). And I have no idea why the code above does not work. :(